iotwifi/iotwifi.go

```diff
@@ -110,23 +110,29 @@ func RunWifi(log bunyan.Logger, messages chan CmdMessage, cfgLocation string) {
 
 	time.Sleep(10 * time.Second)
 
+	// Start supplicant and attempt to connect
 	command.StartWpaSupplicant()
 
-	// Scan
+	// Do a single scan
 	time.Sleep(5 * time.Second)
 	wpacfg.ScanNetworks()
 
 	command.StartDnsmasq()
 
-	// TODO: check to see if we are stuck in a scanning state before
-	// if in a scanning state set a timeout before resetting
+	// monitor for a future connection - shut down AP when it occurs
 	go func() {
 		for {
-			wpacfg.ScanNetworks()
 			time.Sleep(30 * time.Second)
+			if status, err := wpacfg.Status(); err == nil && status["wpa_state"] == "COMPLETED" {
```

Comment:
Should we log this error when it happens? I don't think we handle a non nil error for the Status() at the moment.

Reply:
I think we're safe to ignore it here.  The transient error won't hurt anything since the next loop will catch it.

This same method is invoked when we hit the /status endpoint.  An error in that case will be logged which I think would give us insight if something was borked.